<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDishDialog</class>
 <widget class="QDialog" name="addDishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new dish to the menu</string>
  </property>
  <widget class="QTextBrowser" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter name</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceSpinBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>250</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>250</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
